src/core/ext/filters/client_channel/resolver/dns/dns_resolver_selection.h

```diff
@@ -16,6 +16,10 @@
  *
  */
 
+#include <grpc/support/port_platform.h>
```

Comment:
These should be inside the `#ifndef GRPC_CORE_EXT_FILTERS_CLIENT_CHANNEL_RESOLVER_DNS_DNS_RESOLVER_SELECTION_H` block.

Reply:
done